Apps/WebClient/src/ClientApp/src/components/modal/AddQuickLinkComponent.vue

```diff
@@ -70,16 +70,29 @@ export default class AddQuickLinkComponent extends Vue {
         return (
             (this.enabledQuickLinkFilter.length > 0 &&
                 this.selectedQuickLinks.length > 0) ||
-            (this.showVaccineCard && this.selectedQuickLinks.length > 0)
+            (this.selectedQuickLinks.length > 0 &&
+                (!this.vaccinationStatusModuleEnabled ||
+                    this.preferenceVaccineCardHidden))
         );
     }
```

Comment:
I think this getter can be simplified to just `this.selectedQuickLinks.length > 0`.

Reply:
Good call, I thought that method was way more complicated than it needed to be.  I have simplified it as suggested and also simplified the other code since I don't need the getters any more as they are only used in one place.